bootstrap/integration/checkout_integration_test.go

```diff
@@ -63,7 +63,7 @@ func TestWithResolvingCommitExperiment(t *testing.T) {
 	// But assert which ones are called
 	if experiments.IsEnabled(`git-mirrors`) {
 		git.ExpectAll([][]interface{}{
-			{"clone", "--bare", "--", tester.Repo.Path, matchSubDir(tester.GitMirrorsDir)},
+			{"clone", "--mirror", "--bare", "--", tester.Repo.Path, matchSubDir(tester.GitMirrorsDir)},
```

Comment:
I had to flip all the test expectations, but I think it makes more sense this way, with `--mirror` first`.